src/main/java/org/broadinstitute/hellbender/tools/walkers/readorientation/CollectF1R2Counts.java

```diff
@@ -44,13 +44,16 @@
 @CommandLineProgramProperties(
         summary = "Collect F1R2 read counts for the Mutect2 orientation bias mixture model filter",
         oneLineSummary = "Collect F1R2 read counts for the Mutect2 orientation bias mixture model filter",
-        programGroup = CoverageAnalysisProgramGroup.class
+        programGroup = ShortVariantDiscoveryProgramGroup.class
 )
 
 public class CollectF1R2Counts extends LocusWalker {
     public static final String ALT_DATA_TABLE_LONG_NAME = "alt-table";
+    public static final String ALT_DATA_TABLE_SHORT_NAME = "at";
```

Comment:
I think it's better not to define short names for these.  The long names are short enough.  Note that this changes your wdls, which used the short names.

Reply:
True. Done